stumpy/floss.py

```diff
@@ -2,13 +2,16 @@
 # Copyright 2019 TD Ameritrade. Released under the terms of the 3-Clause BSD license.  # noqa: E501
 # STUMPY is a trademark of TD Ameritrade IP Company, Inc. All rights reserved.
 
+import copy
+from typing import Optional, Tuple, Iterator
+
 import numpy as np
 import scipy.stats
+
 from . import core
-import copy
```

Comment:
I think this `import copy` is required since we need to perform a `copy.deepcopy` later in the file. I'm kind of surprised that the tests didn't fail :(

Reply:
Nevermind, I mis-read this. I see that you are moving it higher up in the imports